apps/main-landing/src/components/products-section/components/prediction-markets-section/prediction-markets-section-actions.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button } from '@idriss-xyz/ui/button';
+import { classes } from '@idriss-xyz/ui/utils';
+import { useMutation } from '@tanstack/react-query';
+import { Controller, useForm } from 'react-hook-form';
+
+type WaitlistInput = {
+  email: string;
+};
+
+export const PredictionMarketsSectionActions = () => {
+  const {
+    control,
+    handleSubmit,
+    reset,
+    formState: { errors },
```

Comment:
there's errors and error msg but I don't see any schema :think:

Reply:
oh I see schema defined inline in controller, I saw you added zod so maybe you can add zod schema there and remove inline validation from Controller

Reply:
or actually you can also just add TODO comment, we can tackle it later